pkg/cmd/build/build.go

```diff
@@ -104,7 +104,7 @@ func (ob *OktetoBuilder) Run(ctx context.Context, buildOptions *types.BuildOptio
 	}
 
 	switch {
-	case IsDepotEnabled():
+	case IsDepotEnabled() && !isDeployOrDestroy:
```

Comment:
Let's add a clear comment above this `switch` explaining why for remote operations we don't use Depot. We will forget about it in a few months